lms/static/js/edxnotes/views/tabs/course_structure.js

```diff
@@ -4,10 +4,12 @@ define([
     'gettext', 'underscore', 'js/edxnotes/views/note_group', 'js/edxnotes/views/tab_panel',
     'js/edxnotes/views/tab_view'
 ], function (gettext, _, NoteGroupView, TabPanelView, TabView) {
+    var view = "Location in Course";
     var CourseStructureView = TabView.extend({
+        view: 'Location in Course',
```

Comment:
@cahrens i guess variable view is defined at line 7 which is https://github.com/edx/edx-platform/pull/8052/files#diff-a5fe99f5253eb4200da6e3bb1ac9289bR7 is't it ?